### PR TITLE
chore: revert "test: temporary disable serverless framework test"

### PR DESCRIPTION
**Pull request by Jolg42**
Reverts prisma/e2e-tests#2298

To merge when the AWS credentials are working again

### CODE REVIEW

File: platforms-serverless/serverless-framework-lambda/run.sh [hunk 1]
```diff
@@ -8,4 +8,8 @@ yarn prisma generate
 
 yarn tsc
 
+# Turn serverless CLI logs to verbose
+# Useful for debugging aws permissions errors (and probably more)
+export SLS_DEBUG=*
```
Comment (Jolg42) [inline platforms-serverless/serverless-framework-lambda/run.sh:13]: I think we can keep this for easier debugging

**Comment by janpio** [comment]
Rerun tests manually?

**Comment by Jolg42** [comment]
@janpio yes already done, and it's good to merge 

Logs are here
library
https://github.com/prisma/e2e-tests/runs/4867991693?check_suite_focus=true
binary
https://github.com/prisma/e2e-tests/runs/4867991433?check_suite_focus=true

**Review by janpio — APPROVED** [review]
Nah I mean the unrelated failing test:
```
  ● Engine Types - default(CLI=library Client=library) › expected(CLI=library, CLIENT=binary) env(PRISMA_CLIENT_ENGINE_TYPE=binary, PRISMA_CLI_QUERY_ENGINE_TYPE=undefined ) 

    thrown: "Exceeded timeout of 200000 ms for a test.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

      310 |   const expected = getExpectedEngineTypes(options)
      311 |   const testName = generateTestName(options, expected)
    > 312 |   return test(testName, async () => {
          |          ^
      313 |     console.log(`Project DIR: ${projectDir}`)
      314 |     // await removePrismaCache()
      315 |     await setupTmpProject(projectDir)

      at utils.ts:312:10
      at step (utils.ts:44:23)
      at Object.next (utils.ts:25:53)
      at utils.ts:19:71
      at Object.<anonymous>.__awaiter (utils.ts:15:12)
      at runTest (utils.ts:286:12)
      at all.test.ts:40:10

Test Suites: 1 failed, 1 skipped, 1 of 2 total
Tests:       1 failed, 108 skipped, 14 passed, 123 total
Snapshots:   15 passed, 15 total
Time:        535.332 s
```

But feel free to ignore if you are confident unrelated

**Pull request merged**